Noting where the tokens come from (#10548)
dotnet/docs: docs/framework/unmanaged-api/metadata/imetadataimport-getinterfaceimplprops-method.md

> ms.author: "mairaw"

Reviewer: given how thoroughoully this was changed, it would be good to update the ms.date value to today's date on line 3

  
 `ptkIface`  
 [out] The metadata token representing the interface that defines the implemented method.  

## Remarks

>  You obtain the value for *iImpl* by calling the **EnumInterfaceImpls** method.

Reviewer: ```suggestion
 You obtain the value for `iImpl` by calling the [EnumInterfaceImpls](imetadataimport-enuminterfaceimpls-method) method.
```

>  For example, suppose a class has an mdTypeDef token value of 0x02000007. And suppose it implements three

Reviewer: ```suggestion
 For example, suppose that a class has an `mdTypeDef` token value of 0x02000007 and that it implements three
```

interfaces whose types have tokens: 

- 0x02000003 (TypeDef)
- 0x0100000A (TypeRef)
- 0x0200001C (TypeDef)

Conceptually, this information is stored into an interface implementation table as:

> | Row Number | Class Token | Interface Token |

Reviewer: ```suggestion
| Row number | Class token | Interface token |
```

|------------|-------------|-----------------|
| 4          |             |                 |
| 5          | 02000007    | 02000003        |
| 6          | 02000007    | 0100000A        |
| 7          |             |                 |
| 8          | 02000007    | 0200001C        |

Recall, the token is a 4-byte value:

> - the lower 3 bytes hold the row number, or RID

Reviewer: ```suggestion
- The lower 3 bytes hold the row number, or RID.
```

> - the upper byte holds the token type – 0x09 for `mdtInterfaceImpl` 

Reviewer: ```suggestion
- The upper byte holds the token type – 0x09 for `mdtInterfaceImpl`.
```

> **GetInterfaceImplProps** will return the information held in the row whose token you provide in the *iImpl* argument. 

Reviewer: ```suggestion
`GetInterfaceImplProps` returns the information held in the row whose token you provide in the `iImpl` argument. 
```
